### PR TITLE
Add Dothill-CSI to driver.md

### PR DESCRIPTION
**Pull request by abuisine**
NONE

### CODE REVIEW

**Comment by abuisine** [comment]
CLA should be ok now

File: book/src/drivers.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [Diamanti-CSI](https://diamanti.com/use-cases/io-acceleration/#csi) | `dcx.csi.diamanti.com` | v1.0 | A Container Storage Interface (CSI) Driver for Diamanti DCX Platform | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion
 [DigitalOcean Block Storage](https://github.com/digitalocean/csi-digitalocean) | `dobs.csi.digitalocean.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for DigitalOcean Block Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion
+[Dothill-CSI](https://github.com/enix/dothill-csi) | `dothill.csi.enix.io` | v1.3 | Generic CSI plugin supporting [Seagate AssuredSan](https://www.seagate.com/fr/fr/support/dothill-san/assuredsan-pro-5000-series/) appliances such as [HPE MSA](https://www.hpe.com/us/en/storage/flash-hybrid.html), [Dell EMC PowerVault ME4](https://www.dell.com/fr-fr/work/shop/productdetailstxn/powervault-me4-series) and others ... | Persistent | Read/Write Single Node | Yes | Expansion
```
Comment (xing-yang) [inline book/src/drivers.md:40]: This url gives me a `404` error: https://github.com/enix/dothill-csi/blob/main/example/storage-class

Comment (abuisine) [reply book/src/drivers.md:40]: solved, thx

**Pull request merged**